docs/firewall_opnsense.rst

```diff
@@ -0,0 +1,780 @@
+and WAN interfaces, that is not covered by the Setup Wizard.
+
+Configure the WAN interface
+'''''''''''''''''''''''''''''
+First, navigate to **Interfaces > [WAN]**. In the **Basic configuration** section,
+check the checkbox labeled **Prevent interface removal**.
+
+In the **Generic configuration**
+section, make sure that the **Block private networks** and **Block bogon networks**
+checkboxes are checked.
+
+Scroll down and click  **Save**, then click **Apply changes** when prompted.
+
+Configure the LAN interface
+'''''''''''''''''''''''''''''
+Next, navigate to **Interfaces > [LAN]**. In the **Basic configuration** section,
+check the checkbox labeled **Prevent interface removal**.
+
+In the **Generic configuration** section, select ``Static IPv4`` in the **IPv4
+Configuration Type** dropdown, and ``None`` in the **IPV6 Configuration Type**
+dropdown.
+
+Scroll down and click **Save**, then click **Apply changes** when prompted.
+
+Configure the OPT1 interface
+'''''''''''''''''''''''''''''
+Next, navigate to **Interfaces > [OPT1]**. In the **Basic configuration** section,
+check the checkbox labeled **Prevent interface removal**.
+
+In the **Generic configuration** section, select ``Static IPv4`` in the **IPv4
+Configuration Type** dropdown, and ``None`` in the **IPV6 Configuration Type**
+dropdown.
+
+Scroll down. In the **Static IPv4 Configuration** section, enter the *Application
+Gateway* IP address and routing prefix (``10.20.2.1`` and ``24`` if you are using
+the recommended values).
+
+Click **Save**, then click **Apply changes** when prompted.
+
+Configure the OPT2 interface
+'''''''''''''''''''''''''''''
+Finally, navigate to **Interfaces > [OPT2]**. In the **Basic configuration** section,
+check the checkbox labeled **Prevent interface removal**.
+
+In the **Generic configuration** section, select ``Static IPv4`` in the **IPv4
+Configuration Type** dropdown, and ``None`` in the **IPV6 Configuration Type**
+dropdown.
+
+Scroll down. In the **Static IPv4 Configuration** section, enter the *Monitor
+Gateway* IP address and routing prefix (``10.20.3.1`` and ``24`` if you are using
+the recommended values).
+
+Click **Save**, then click **Apply changes** when prompted.
+
+Configure Firewall Aliases
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In order to simplify firewall rule setup, the next step is to configure aliases
+for hosts and ports referred to in the rules.
+
+To start, first navigate to **Firewall > Aliases**. You should see some system-defined
+aliases as shown below:
+
+|OPNSense - Alias Start|
+
+Click the **+** button to add new aliases. You should add the aliases defined in
+the table below (assuming recommended values for IP addresses):
+
+.. list-table:: Firewall Aliases
+   :header-rows: 1
+
+   * - Name
+     - Type
+     - Content
+
+   * - admin_workstation
+     - Host(s)
+     - 10.20.1.2
+
+   * - app_server
+     - Host(s)
+     - 10.20.2.2
+
+   * - external_dns_servers
+     - Host(s)
+     - 8.8.8.8, 8.8.4.4
```

Comment:
When I typed `8.8.8.8, 8.8.4.4`, OPNsense parsed this string as (quotation marks used to prevent GitHub from swallowing significant leading whitespace):

1. `"8.8.8.8"`
2. `" 8.8.4.4"`

and rejected (2) as invalid (which it is).  It might be worth giving explicit instructions for this alias's content, something like:

> Enter `8.8.8.8`, press &lt;Enter&gt;, then `8.8.4.4`, and press &lt;Enter&gt; again.

Or just `8.8.8.8,8.8.4.4` without a space following the comma.

Reply:
fixed.

Reply:
Actually, after looking at this again, I've opted to change the styling on those alias values and preserve the readability of the comma separation in the table itself. I don't want to get into a level of granularity in the instructions where we're dictating every keystroke, as the docs are intended for a relatively technical audience - instead the goal is to make sure they have the correct information on hand and a decent idea of the path through the GUI.

Reply:
Makes sense, and I like this compromise.  The new rendering—

> `8.8.8.8`, `8.8.4.4`

—more closely matches how the parsed values will appear in the OPNsense interface.